frontend/src/components/common/icon/styles.module.scss

```diff
@@ -0,0 +1,6 @@
+.icon {
+  display: inline-block;
+  overflow: visible;
+  font-size: inherit;
+  fill: currentcolor;
```

Comment:
```suggestion
  fill: currentColor;
```

Reply:
![image](https://user-images.githubusercontent.com/61088675/183696064-29188535-18bb-4960-9672-48684ac4e1e4.png)
How can I fix this? Seems like it is related to eslint